src/client/common/components/base-network-view/config/index.js

```diff
@@ -38,6 +38,13 @@ const toolbarButtons = [
     menuId: 'fileDownloadMenu',
     description: 'Download options'
   },
+  location.pathname.includes('interactions')? {
```

Comment:
See https://github.com/PathwayCommons/app-ui/blob/master/src/client/features/paint/index.js#L24

for how to inject buttons into the base network component